src/Components/ArticlePage.jsx

```diff
@@ -3,45 +3,48 @@ import Button from 'react-bootstrap/Button'
 import {useParams, Link } from 'react-router'
 import {useEffect, useState}from 'react'
 import axios from 'axios'
-import CommentCard from './CommentCard'
+import CommentList from './CommentList';
 
 function ArticlePage (){
     const {article_id} = useParams();
     const [article, setArticle] = useState([])
-    const [comments, setComments] = useState([])
+    const [votes, setVotes] = useState(0)
     useEffect(() => {
         const getArticleById = async () => {
             const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}`)
             setArticle(data.article)
+            setVotes(data.article.votes)
+            console.log(data.article)
+          
```

Comment:
It's possible to do this without adding in an extra state for `votes, setVotes` by using the `article` state you already have and updating that - this is something you could possibly look at refactoring if you have time later

Reply:
also don't forget to remove console logs 🧹 ✨